public/votes.cfg

```diff
@@ -1,44 +1,35 @@
 # public/votes.cfg
+# Get the maps from https://twelo.pages.dev/maps
+# You can also copy some of the text formatting for the votes here: https://github.com/PathosEthosLogos/tw/blob/master/DM_Atlanta(most%20recent)
 
 clear_votes
 
+add_vote "[===] game [===]" "sv_map"
+
+add_vote "restart" "restart 10"
+add_vote "[ ] enable ready mode" "sv_player_ready_mode 1; say 0.7 players can now pause the game with f1 ready_change. And 0.6 players with the /ready chat command."
+add_vote "[ ] disable ready mode" "sv_player_ready_mode 0; say the ready_change command no longer allows pausing the game"
+add_vote "pause" "pause_game"
+add_vote "shuffle teams" "shuffle_teams"
+add_vote "swap teams" "swap_teams"
+add_vote "reload server" "reload"
+
 add_vote "[===] maps [===]" "sv_map"
 
-add_vote "dm1" "sv_gametype dm;sv_map dm1; sv_spectator_slots 62;sv_scorelimit 10"
-add_vote "ctf1" "sv_gametype ctf;sv_map ctf1; sv_spectator_slots 60; sv_scorelimit 600"
-add_vote "ctf2" "sv_gametype ctf;sv_map ctf2; sv_spectator_slots 48; sv_scorelimit 600"
-add_vote "ctf3" "sv_gametype ctf;sv_map ctf3; sv_spectator_slots 58; sv_scorelimit 600"
-add_vote "ctf4" "sv_gametype ctf;sv_map ctf4; sv_spectator_slots 58; sv_scorelimit 600"
-add_vote "ctf5" "sv_gametype ctf;sv_map ctf5_spikes; sv_spectator_slots 48; sv_scorelimit 600"
-add_vote "ctf6" "sv_gametype ctf;sv_map ctf6"
-add_vote "ctf7" "sv_gametype ctf;sv_map ctf7"
-add_vote "ctf8" "sv_gametype ctf;sv_map ctf8"
-add_vote "ctf4_old" "sv_map ctf4_old"
-add_vote "ctf_ambiance" "sv_map ctf_ambiance"
-add_vote "ctf_duskwood" "sv_map ctf_duskwood"
-add_vote "ctf_lava" "sv_map ctf_lava"
-add_vote "ctf_red_afternoon" "sv_map ctf_red_afternoon"
-add_vote "ctf_take" "sv_map ctf_take"
-add_vote "ctf_tantum" "sv_map ctf_tantum"
-add_vote "ctf_moon" "sv_map ctf_moon"
-add_vote "ctf_corp" "sv_map ctf_corp"
-add_vote "ctf_mine" "sv_map ctf_mine"
-add_vote "ctf_planet" "sv_map ctf_plan_egg"
-add_vote "ctf_katm" "sv_map ctf_katm"
-add_vote "ctf_aurochs" "sv_map ctf_aurochs"
-add_vote "desertcamp" "sv_map desertcamp"
-add_vote "ctf4_old_3n3test" "sv_map ctf4_old_3n3test"
+add_vote "dm1" "sv_gametype dm; sv_map dm1; sv_spectator_slots 62; sv_scorelimit 10"
+add_vote "dm1mirror" "sv_gametype dm; sv_map dm1mirror; sv_spectator_slots 62; sv_scorelimit 10"
```

Comment:
this map is not in this map is not in the [map pool](https://github.com/ddnet-insta/maps-06)